### DISCUSSION TITLE
Convert Multiple Pages from PDF to CSV

### DISCUSSION THREAD

**Discussion by ByndGalaxiesS** [Stirling-Tools/Stirling-PDF#2000] [Q&A]
I often get generated PDFs that I cannot download in any other format that have tables in them. I'd like to use the "Convert to CSV" optikon, and it seems to work one page at a time to download. But my documents are often 600+ pages for the projects I am working on. Is this tool a viable solution for that need? I've got it installed on a local container but can't seem to wrap my head around the multiple pages issue.

**Comment by Frooodle** [comment]
Would each page be a new table or same table?

**Reply by ByndGalaxiesS** [reply]
The way these are formatted, sadly, is that each page looks like its own table.

Ideally, if there was an export all pages to CSV on sheet 1 (excel), I could write a vb script to combine them into one table.

The way I am interpretting the stirling tool right now is that I would have to export each individual page and open 600+ pages to slowly combine them.

As a side note about the documentation, it has HIPPA details that I cannot export to other public solutions for conversion.
